### DISCUSSION TITLE
Does not respawn after selecting a team

### DISCUSSION THREAD

**Discussion by latelyjustwannadream** [hedgefog/cs-halloween-mod#73] [Q&A] [answered]
After the first player selects a team and is revived.
The second player who chose a team has not been revived

**Comment by hedgefog** [comment]
Hello! Thank you for your bug report!
Please use the **Issues** section to report bugs.  The **Discussions** section is more like a forum for communication and sharing custom content.

Respawn should generally work fine, but I acknowledge there could be issues. Could you let me know which version of Halloween Mod you're using, and whether you're using the `reapi` or `vanilla` build?"

**Reply by latelyjustwannadream** [reply]
hello
ReAPI - 5.24.0.300
Amxx - 1.10
Mode ver - 6.4.2
respawn occurs at the end of the round
in hw_controls, I replaced the command to open the menu, could this have led to this error?
it's just that the server menu (multimode server) opens on M (chooseteam)
and at the same time I made a simple command choice

**Reply by hedgefog** [reply]
Depends on how you did it :)

**Reply by latelyjustwannadream** [reply]
in hwn_controlls register_clcmd("modemenuhw", "OnClCmd_ChooseTeam");
only "chooseteam" is responsible for opening the menu
and "changeteam" is responsible for choosing a team
changeteam.sma code -> https://pastebin.com/Dxak1ZHc
I don't understand much about scripting, I found the code and just redid it a little:)

**Comment by hedgefog** [comment, marked as answer]
It looks like this might have an impact on the respawn logic. Please, try with the original plugin.
